core/selection.js

```diff
@@ -263,7 +280,7 @@ class Selection {
     }
   }
 
-  setNativeRange(startNode, startOffset, endNode = startNode, endOffset = startOffset, force = false) {
+  setNativeRange(startNode, startOffset, endNode = startNode, endOffset = startOffset, force = false, reversed = false) {
```

Comment:
It is not necessary to add a parameter since there startNode and endNode can just be reversed.